bazel-build.el

```diff
@@ -24,6 +24,17 @@
 ;;
 ;;; Code:
 
+(defgroup bazel-build nil
+  "Package for building and running code using Bazel."
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+(defcustom bazel-build-bazel-program "bazel"
+  "Name of the 'bazel' program for 'bazel-build' and 'bazel-test'."
```

Comment:
Nit: Quote Lisp symbols in docstrings `` `like-this' `` or `‘like-this’`. See https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html.

Reply:
SG, thanks, please ping this thread once you're ready.